### DISCUSSION TITLE
Purchasing/Gifting Github Copilot

### DISCUSSION THREAD

**Discussion by Arifursdev** [community/community#65786] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It would be fantastic to have the option to purchase a coupon code or gift card that allows the gifting of GitHub Copilot to someone.

**Comment by Akash1134** [comment]
Hi @Arifursdev ,

Thank you for your feedback. Currently, it is not possible to purchase a coupon code or gift card that allows you to gift GitHub Copilot.

**Reply by devinatkin** [reply]
This is especially sad given christmas is coming up. I thought I had just come up with the perfect gift for my brother. Even if it were a generic "github giftcard" that would make a great gift.

**Reply by xuelink** [reply]
@Akash1134 Not only christmas but also many many reasons we need it, please think about it again
I want to make giveaway monthly for my outstanding contributors at my [organization](https://github.com/langx)

**Comment by praveenc** [comment]
+1 - I understand needing a Credit Card to purchase CoPilot for Business Users. But for individual accounts this doesn't make sense. Not all users have a CC account and makes it hard for gifting.

What would be really beneficial is to make Gift cards for CoPilot and allow users to purchase it off of stores like Amazon, Walmart, Costco etc.,

This'd be great for gifting to family members, members in a small team or large. Sad this doesn't exist today.

**Comment by chris-amaya** [comment]
this is so sad, I was hopping I could gift a subscription in christmas, does anyone know another way to gift a subscription? probably with some 1 time only cc? 🤔

**Comment by perrauo** [comment]
I would love this feature too.

**Comment by MusicMakerOwO** [comment]
I've been looking to do giveaways to a large community, I've been loving copilot and want to give it away to people to try but I can't just give out credit card information, and just giving a pre-paid visa doesn't really feel the same to me, nor does it work internationally. Please add an option to gift a month or two <3

**Comment by blackysky** [comment]
That would indeed be a very important feature!

**Comment by xoofx** [comment]
I would like to offer copilot to a friend and I can't do it. Please make it happen

**Comment by iKinoo** [comment]
+1 (my brother's birthday is soon)

**Comment by moty66** [comment]
As developers, we understand how GitHub Copilot can boost productivity, many of us want to share this amazing tool with colleagues and friends. 

The absence of an option to gift subscriptions is a significant gap. Implementing this feature would not only strengthen the community but also make GitHub Copilot more accessible, allowing everyone to harness the power of AI in coding.

+1

**Comment by notasandworm** [comment]
i would love to gift copilot sub to my brother too

**Comment by ubogdan** [comment]
It will be a nice feature. i was thinking to offer as a Christmas gift.

**Comment by MochSM** [comment]
+1. I have coding community here and would like to gift them all copilot.

**Comment by kholy** [comment]
I would love to buy a subscription for my younger cousin who is still studying and not making enough money to purchase it

**Reply by laraibg786** [reply]
If your cousin is an active student, there is always option for the [education benefits](https://github.com/education) from Github.

**Comment by 24by36** [comment]
+1 Need a few for coworkers. If they get hooked to it, there's no "GOING BACK" ..  Could cover PRO subscription for a whole year.

**Comment by raymondlowe** [comment]
+1 to this should be an option! It would be great to gift people - including FOSS maintainers as well as your own contacts.

**Comment by ghost** [comment]
I was searching for my best friend and came to this post. This is unlucky that Github haven't handled yet for last 3 years...

**Comment by Gosolceser1** [comment]
Add gifting capabilities, please. I would like to give a present to try GitHub Copilot Pro+. To my friends

**Comment by mishani0x0ef** [comment]
It's frustrating that we still don't have gifting capabilities.

It would be a great gift for fellow devs.

**Comment by rictic** [comment]
Someone on my list this morning is getting Claude even though I know he'd like Copilot better, because I can gift him Claude but I can't gift him Copilot.

**Comment by RedouaneBALI** [comment]
Great idea !
